Project-Hotpot/Project-Hotpot/Managers/SpotifyManager.swift

```diff
@@ -193,12 +202,13 @@ extension SpotifyManager {
                                        "Accept": "application/json"]
         
         makeSpotifyRequest(request: request, completion: completion)
-       
+        
     }
     
-    func fetchAccessToken(completion: @escaping ([String: Any]?, Error?) -> Void) {
-        guard let url = URL(string: "https://accounts.spotify.com/api/token") else {
-            return
+    func fetchAccessToken(completion: @escaping (_ result: Result<[String: Any], Error>) -> Void) {
+        guard let url = URL(string: "https://accounts.spotify.com/api/token")
+        else {
+            return completion(.failure(CustomError.unexpected))
```

Comment:
You know, I don't believe I've ever seen `return completion(...` for `Void` return methods... usually it's `completion(...` followed by `return` on the following line.

I don't think it hurts anything? Who knows, maybe you'll start a trend! Simply calling it out as non-standard, I have yet to make up my mind how I feel about this personally 😅